projects/angular/src/emphasis/_labels.clarity.scss

```diff
@@ -281,39 +141,18 @@
     @include generateStatusLabelStyles(danger);
 
     & > .badge {
-      margin: 0 (-1 * variables.$clr_baselineRem_0_375) 0 variables.$clr_baselineRem_0_25;
-      //-0.375 is -9px. 12px initial label padding plus a negative margin of 9
+      margin: 0;
```

Comment:
Probably this setting to 0 is obsolete, use cds tokens instead of global variables, if so